selene_sdk/targets/genomic_features.py

```diff
@@ -133,7 +135,7 @@ def _get_feature_data(chrom, start, end,
         `i`th feature is positive, and zero otherwise.
 
     """
-    rows = get_feature_rows(chrom, start, end)
+    rows = get_feature_rows(chrom, start, end, strand)
```

Comment:
did you change this method definition somewhere so that it also optionally takes in `strand` as an argument? I might have missed it